code/__HELPERS/unsorted.dm

```diff
@@ -1799,7 +1799,7 @@ var/mob/dview/dview_mob = new
 			/obj/item/cartridge = "PDA_CART",
 			/obj/item/clothing/head/helmet/space = "SPESSHELMET",
 			/obj/item/clothing/head = "HEAD",
-			/obj/item/clothing/under = "UNIFORM",
+			/obj/item/clothing/under = "7IFORM",

```

Comment:
`7IFORM`? That's interesting. Mind explaining this and why this isn't `UNIFORM` anymore (just to sate my curiosity)?

Reply:
Probably mistyped, whoooooops

Reply:
That was quick, thanks for getting back to me haha

Reply:
wot...

Reply:
This needs to be corrected